Update SaritasaTools code styles

namnguyen-saritasa opened the pull request:
`.editorconfig` has been updated

https://saritasa.atlassian.net/browse/SN-744

cycastic-cumberland committed:
Updated `.editorconfig` SN-744

cycastic-cumberland committed:
Updated `.nuspec.template` SN-744

cycastic-cumberland committed:
Updated `README.md` SN-744

ko-vasilev commented:
@namnguyen-saritasa there is another small update, can you please apply it as well? https://github.com/saritasa-nest/saritasa-dotnet-boilerplate-web/pull/32/files#diff-5b9c60ecca140eea301d5a21f8e55fa27d1c0eb8c4169567cffdbe78437a21b9

cycastic-cumberland committed:
Updated editorconfig SN-744

cycastic-cumberland committed:
Added XML docs to non-test source files SN-744

namnguyen-saritasa commented:
@ko-vasilev I have modified the rule for `SA1600` in `.editorconfig` and added missing XML docs to non-test methods. Should I add docs to the remaining methods?

ko-vasilev commented on src/Saritasa.Tools.Common/Utils/DateTimeUtils.cs:
```diff
@@ -277,7 +277,7 @@ public static double GetDiff(DateTime target1, DateTime target2, DateTimePeriod
         }
 
-        throw new ArgumentException(nameof(period));
+        throw new ArgumentException(null, nameof(period));
```
```suggestion
        throw new ArgumentException("Unsupported period value.", nameof(period));
```

cycastic-cumberland committed:
Updated `GetDiff`'s exception message SN-744
